app/views/docs/authentication.phtml

```diff
@@ -691,76 +893,646 @@ let session = try await account.get()</code></pre>
+
+val account = Account(client)
+
+val response = account.createJWT()</code></pre>
+    </li>
+    <li>
+        <h3>Apple</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+let account = Account(client)
+
+let jwt = try await account.createJWT()</code></pre>
+    </li>
+    <li>
+        <h3>GraphQL</h3>
+        <div class="ide" data-lang="graphql" data-lang-label="GraphQL">
+            <pre class="line-numbers"><code class="prism language-graphql" data-prism>mutation {
+    accountCreateJWT {
+        jwt
+    }
+}</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>Your server application can use the JWT to act on behalf of the user by creating a <code>Client</code> instance with the JWT for each request it receives with a JWT.</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Node.js</h3>
+        <div class="ide margin-top-small" data-lang="nodejs" data-lang-label="Node.js SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const { Client } = require('node-appwrite');
+
+const client = new Client();
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2')                // Your project ID
+    .setJWT('eyJJ9.eyJ...');                    // Your secret JSON Web Token</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>PHP</h3>
+
+        <div class="ide margin-top-small" data-lang="php" data-lang-label="PHP SDK">
+            <pre class="line-numbers"><code class="prism language-php" data-prism>use Appwrite\Client;
+
+$client = new Client();
+
+$client
+    ->setEndpoint('https://[HOSTNAME_OR_IP]/v1')             // Your API Endpoint
+    ->setProject('5df5acd0d48c2')                            // Your project ID
+    ->setJWT('eyJ9.eyJ...'); // Your secret JSON Web Token</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Python</h3>
+        <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
+            <pre class="line-numbers"><code class="prism language-python" data-prism>from appwrite.client import Client
+
+client = Client()
+
+(client
+  .set_endpoint('https://[HOSTNAME_OR_IP]/v1')              # Your API Endpoint
+  .set_project('5df5acd0d48c2')                             # Your project ID
+  .set_jwt('eyJ9.eyJ...')   # Your secret JSON Web Token
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Ruby</h3>
+        <div class="ide margin-top-small" data-lang="ruby" data-lang-label="Ruby SDK">
+            <pre class="line-numbers"><code class="prism language-ruby" data-prism>require 'appwrite'
+
+include Appwrite
+
+client = Client.new
```

Comment:
For consistency with other server examples lets create the client and call the setters separately:

```suggestion
client = Client.new

client
```

Also applies for Kotlin + Swift

Reply:
Done